todo_app/app.py

```diff
@@ -63,20 +74,23 @@ def load_user(user_id):
     def validate_user_role(func):
         @wraps(func)
         def wrapTheFunction(*args, **kwargs): 
-            if (not loginDisabled and current_user.role == "reader"):    
+            if (not loginDisabled and current_user.role == "reader"):
+                app.logger.info(f"Insufficient User Rights")
```

Comment:
Is this worthy of being a `.warn` rather than `.info`? The answer might depend on our system; if we expect users to regularly accidentally trigger actions with insufficient rights then info might make more sense, but if we've hidden the relevant actions then this might be indicative of an underlying issue in which case bumping it up might make sense